src/core/viewrenderer/BaseViewRenderer.tsx

```diff
@@ -28,6 +28,7 @@ export interface ViewRendererProps<T> {
     extendedState?: object;
     internalSnapshot?: object;
     layoutProvider?: BaseLayoutProvider;
+    onItemLayout?: (layout: Dimension, index: number) => void;
```

Comment:
dev may also be interested in x, y values

Reply:
Done